test/microapps-ui-xcm/microapps-ui-main.withdraw.test.ts

```diff
@@ -158,6 +160,13 @@ describe("Microapps UI withdraw modal tests", () => {
       TransactionType.Withdraw,
     );
     expect(isModalWaitingForSignVisible).toBeTruthy();
+    await waitForMicroappsActionNotification(
```

Comment:
why this?

Reply:
We need to wait several blocks to receive approval from Kusama parachain

Reply:
but we are not sending any token, looks like this test ( from what I can understand )  only opens the modal.

Reply:
no no , it withdraws 1 KSM

Reply:
all right, my bad